honeybee_radiance_command/options/rcontrib.py

```diff
@@ -54,8 +54,9 @@ def _on_setattr(self):
         instance in rtrace option collection -ti and -te are exclusive. You can include a
         check to ensure this is always correct.
         """
-        RtraceOptions._on_setattr(self)
-        # -r and -fo cannot both be True.
+        # -i and -I cannot both be True.
+        assert not (self.i == True and self.I == True), \
+            'You can either set -i or -I to True not both.'
```

Comment:
You probably want to move this after line 60 and only execute the check once the class has been initialized. Other than that it looks good to me!